githubscript/src/githubscript/script.py

```diff
@@ -21,6 +21,7 @@ async def async_main(config, task):
     projects = config["github_projects"].keys()
     if project in projects:
         project_config = config["github_projects"][project]
+        project_config.setdefault("scope_project", project)
```

Comment:
I don't think we need this; we only need to verify the project matches the scope if we're using a glob.